tests/virt_autotest/reboot_and_wait_up_normal4.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: virt_autotest: the initial version of virtualization automation test in openqa, with kvm support fully, xen support not done yet
+# Maintainer: alice <xlai@suse.com>
+
+use strict;
+use warnings;
+use testapi;
+use base "reboot_and_wait_up";
+
+sub run {
+    my $self    = shift;
+    my $timeout = 120;
+    $self->reboot_and_wait_up($timeout);
+}
+
+sub test_flags {
+    return {fatal => 1};
+}
+
+1;
+
```

Comment:
All files tests/virt_autotest/reboot_and_wait_up_normal[1-4] look very similar. I suggest to use symlinks for all instead